assets/js/modules/analytics/common/account-create.js

```diff
@@ -0,0 +1,163 @@
+/**
+ * AccountCreate component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState, Fragment, useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../../components/button';
+import ProgressBar from '../../../components/progress-bar';
+import { trackEvent } from '../../../util';
+import TimezoneSelect from './timzezone-select';
+import AccountField from './account-field';
+import PropertyField from './property-field';
+import ProfileField from './profile-field';
+
+// import Data from 'googlesitekit-data';
+// const { dispatch, select } = Data;
+
+const AccountCreate = () => {
+	// const { createAccount } = dispatch( STORE_NAME );
+	// const isDoingCreateAccount = select( STORE_NAME ).isDoingCreateAccount();
```

Comment:
@aaemnnosttv I may need some help using this store element to control the 'submitting' state here. Previously I tracked this in component state, however I'm trying to use the store instead.

I've tried using `withSelect`, however the select element passed to the callback there doesn't have access to the correct store. Using `select` from Data directly does work, but only runs once (eg the component doesn't update when the store value changes).

I tried something like this:
```js
const isDoingCreateAccount = useSelect(
		( select ) => {
			return select( STORE_NAME ).isDoingCreateAccount();
		},
		[]
	);
```

Reply:
Right! Thanks for clarifying.

Reply:
Added this and switched to `useDispatch` as well (commented out for now).